README.md

```diff
@@ -190,5 +190,25 @@ Or you can run all the tests by running this:
 **Please be aware that some tests in the test_column2Vec 
 module may take a long time.**
 
+### Static tests
+For static tests, we use pylint and black.
+#### Pylint
+You can run it by using this command:
+```bash
+pylint $(git ls-files '*.py')
```

Comment:
Here, you follow examples from generic to specific ones. Well done.
Could you do the same for pytest usage above?

Reply:
Fixed